WMIAdventure/backend/WMIAdventure_backend/battle/businesslogic/Player.py

```diff
@@ -13,15 +13,17 @@ class Player:
     Has statistics that represent its current state and its card deck.
     """
 
-    def __init__(self, id: int, deck: Deck):
+    def __init__(self, id: int, deck: Deck, level: int):
```

Comment:
No właśnie, jakbyśmy wyliczali exp po walce w zupełnie innym miejscu, to nie musielibyśmy tego przechowywać w playerze. Ten level nie jest nigdzie potrzebny poza właśnie wyliczaniem skutków walki